ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -862,11 +862,13 @@ let empty_pool_uefi_certificates =
 let update_livepatch_guidance =
   {
     description=
-      "Replace reboot_host_on_livepatch_failure in host.pending_guidances with \
-       reboot_host_on_kernel_livepatch_failure and \
+      "1. Replace reboot_host_on_livepatch_failure in host.pending_guidances \
```

Comment:
This description seems too long to be in log. Could it be a one-line sentence and change this paragraph to comment?